src/instructlab/cli/data/generate.py

```diff
@@ -165,6 +165,20 @@
 @click.option(
     "-dt", "--detached", is_flag=True, help="Run ilab data generate in the background"
 )
+@click.option(
+    "--student-model-id",
+    # help="ID of the custom model you want to use as the student model.",
+    default=None,
+    type=click.STRING,
+    show_default=True,
+)
+@click.option(
+    "--teacher-model-id",
```

Comment:
same here for default

Reply:
@cdoern We should leave it blank for now and figure out the best way for this later.